modules/monitoring-configuring-a-local-persistent-volume-claim.adoc

```diff
@@ -161,8 +161,13 @@ data:
           storageClassName: *local-storage*
           resources:
             requests:
-              storage: *40Gi*
+              storage: *10Gi*
 ----
++
+[NOTE]
+====
+Storage requirements for `thanosRuler` depend on the number of rules evaluated and how many samples each rule generates.
```

Comment:
When referring to a literal value that is wrapped in backticks, we should specify the object of that description, such as parameter, resource, or object. In this case, I believe it looks like Thanos Ruler is a [component](https://docs.openshift.com/container-platform/4.8/monitoring/configuring-the-monitoring-stack.html#configurable-monitoring-components_configuring-the-monitoring-stack). 

Here's more info on our docs guidelines for referring to object references: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

So my suggestion here would be to write this as "...requirements for the `thanosRuler` component depend...."

Reply:
For readability, I might also suggest `s/rules evaluated/rules that are evaluated/`.